beluga_amcl/src/ndt_amcl_node_3d.cpp

```diff
@@ -245,8 +254,20 @@ beluga::NDTSensorModel<NDTMapRepresentation> NdtAmclNode3D::get_sensor_model() c
   const auto map_path = get_parameter("map_path").as_string();
   RCLCPP_INFO(get_logger(), "Loading map from %s.", map_path.c_str());
 
+  //return beluga::NDTSensorModel<NDTMapRepresentation>{
+  //    params, beluga::io::load_from_hdf5<NDTMapRepresentation>(get_parameter("map_path").as_string())};
+
+  // Store the map (sparse_value_grid?)
+  auto map =
+    beluga::io::load_from_hdf5<NDTMapRepresentation>(get_parameter("map_path").as_string());
+
+  // Get the markers
+  beluga_ros::msg::MarkerArray obstacle_markers = beluga_ros::assign_obstacle_map(map);
+  // Publish the message
+  map_visualization_pub_->publish(obstacle_markers);
```

Comment:
Avoid over-commenting the code, you can make a comment of what a piece of code is doing, but not a comment per-line

```suggestion
  // Publish markers for map visualization
  beluga_ros::msg::MarkerArray obstacle_markers = beluga_ros::assign_obstacle_map(map);
  map_visualization_pub_->publish(obstacle_markers);
```